Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -41,6 +41,11 @@ struct DashboardView: View {
                 }
                 .listStyle(.inset)
             }
+            .overlay(Group {
+                if viewModel.events.isEmpty {
+                    Text("Add your first maintenance")
+                }
+            })
```

Comment:
I'm going to merge this because this is a small change, but note that this doesn't need to be in a `Group` also be aware of your spacing, this is how this code should be spaced properly
```swift
.overlay {
    if viewModel.events.isEmpty {
        Text("Add your first maintenance")
    }
}
```

Reply:
Sure. I will keep that in mind.